### PR TITLE
Some more local network access are removed

### PR DESCRIPTION
**Pull request by annando**
Some more stuff to avoid network traffic:

- We now don't poll local accounts anymore on `OnePoll`
- We fetch the last update directly via database
- Avatar pictures from local accounts are fetched directly via database

### CODE REVIEW

File: src/Worker/OnePoll.php [hunk 1]
```diff
@@ -103,6 +103,15 @@ public static function execute($contact_id = 0, $command = '')
 			return;
 		}
 
+		// Don't poll local contacts
+		if (Contact::isLocalById($contact['id'])) {
```
Comment (MrPetovan) [inline src/Worker/OnePoll.php:107]: At this point we have the contact URL in `$contact['url']`, wouldn't it be cheaper to perform the local check directly on it?

Comment (annando) [reply src/Worker/OnePoll.php]: I replaced this function with another one.

Comment (annando) [reply src/Worker/OnePoll.php]: @MrPetovan is this solution okay for you?

Comment (MrPetovan) [reply src/Worker/OnePoll.php]: Sorry I miss your comment.

**Pull request merged**